addons/interact_menu/functions/fnc_userActions_addHouseActions.sqf

```diff
@@ -36,12 +36,12 @@ if ((vehicle ACE_player) != ACE_player) exitWith {};
         // Prevent Rare Error when ending mission with interact key down:
         if (isNull ace_player) exitWith {};
 
-        //Make the common case fast (cursorTarget is looking at a door):
-        if ((!isNull cursorTarget) && {cursorTarget isKindOf "Static"} && {!(cursorTarget in _housesScaned)}) then {
-            if (((count (configOf cursorTarget >> "UserActions")) > 0) || {(count (getArray (configOf cursorTarget >> "ladders"))) > 0}) then {
-                _housesToScanForActions = [cursorTarget];
+        //Make the common case fast (cursorObject is looking at a door):
```

Comment:
Why `cursorObject` instead of `cursorTarget`.

Reply:
`cursorTarget` is more similar to how scroll-wheel menu actions are detected. I don't see the reason for the change.

Reply:
I think cursorTarget needs the player to `knowsAbout` the target
cursorObject just returns what is in the crosshairs, which is probably the better choice

Reply:
`cursorObject` is a lot more precise tho. Can make it so you can need to look "more directly" at object to show the interactions.

Reply:
Reasoning for change to `cursorObject` was because of `knowsAbout` requirement of `cursorTarget`. I don't mind either way.